From b55788f355243df0736e6312c6125cc1dec6ae1b Mon Sep 17 00:00:00 2001
From: Ken Ma <make@marvell.com>
Date: Tue, 21 Jun 2016 17:31:03 +0800
Subject: [PATCH 1210/1240] config: a3700: remove trusted boot options

- Options MVEBU_SECURE_BOOT and MVEBU_IMAGE_ENCRYPT used to be for ATF
  build only, there is no reference in u-boot at all
- Now ATF has added these two options in build command, instead of get
  it from u-boot .config file
- Remove these two options in u-boot

Change-Id: Iaa1c710c9038e5c53da485c4e53c6457e6cf67d0
Signed-off-by: Ken Ma <make@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/30596
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Hua Jing <jinghua@marvell.com>
Reviewed-by: Wilson Ding <dingwei@marvell.com>
---
 arch/arm/cpu/mvebu-common/Kconfig | 29 -----------------------------
 1 file changed, 29 deletions(-)

diff --git a/arch/arm/cpu/mvebu-common/Kconfig b/arch/arm/cpu/mvebu-common/Kconfig
index 38cba54..323246c 100644
--- a/arch/arm/cpu/mvebu-common/Kconfig
+++ b/arch/arm/cpu/mvebu-common/Kconfig
@@ -1,34 +1,5 @@
 
 
-config MVEBU_SECURE_BOOT
-	bool "Trusted Boot Image format"
-	depends on MVEBU
-	default n
-	help
-	  Instructs the u-boot to create a trusted
-	  boot image for MVEBU device. The trusted
-	  boot image will boot normally on devices
-	  that have secure mode option turned OFF
-	  in their eFUSE. Once the secure boot mode
-	  is selected by eFuse, the trusted boot
-	  image will be verified according to secure
-	  boot flow algorithm described in details
-	  in appropriate device specification.
-
-config MVEBU_IMAGE_ENCRYPT
-	bool "Encrypt Boot Image in Trusted Boot mode"
-	depends on MVEBU_SECURE_BOOT
-	default n
-	help
-	  Instructs the u-boot to use AES-256 encryption
-	  when building image for the trusted boot mode on
-	  MVEBU device.
-	  The AES-256 symmetric key should be burned in
-	  the device eFuse for allowing BootROM to decrypt
-	  the boot image upon startup.
-	  Refer to the appropriate device specification
-	  and eFuse module help for detailes about this feature
-
 config MVEBU_SPL_DIFFRENT_BASE_ADDR
 	bool "Use SPL different registers base address"
 	depends on MVEBU && TARGET_ARMADA_38X
-- 
1.9.1

